playbooks/scenario18_create_cce.yaml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Scenario 18 - CCE Lifecycle
+  hosts: localhost
+  vars:
+    prefix: scenario18-
+  tasks:
+
+    - name: Set facts
+      set_fact:
+        test_keypair_name: "{{ (prefix + 'keypair') }}"
+        vpc_name: "{{ ( prefix + 'router' ) }}"
+        nw_name: "{{ ( prefix + 'network' ) }}"
+        sn_name: "{{ ( prefix + 'subnet' ) }}"
+        cce_name: "{{ ( prefix + 'cluster' ) }}"
+        cce_node: "{{ ( prefix + 'node' ) }}"
+
+    - debug:
+        msg: "Using prefix {{ prefix }}"
+
+    - name: List Keypairs
+      script: list_keypairs.py
+
+    - name: Ensure .ssh exists
+      file:
+        path: "~/.ssh"
+        state: directory
+
+    - name: Create Keypair
+      include_role:
+        name: opentelekomcloud.keypair
+      vars:
+        keypair_name: "{{ test_keypair_name }}"
+
+    - block:    
+        - name: Create VPC (Router + Net + Subnet)
+          include_role:
+            name: opentelekomcloud.vpc
+          vars:
+            router_name: "{{ vpc_name }}"
+            network_name: "{{ nw_name }}"
+            subnet_name: "{{ sn_name }}"
+
+        - name: List all Routers
```

Comment:
Is this required? There is not need from my POV to list neutron components in this scenario.